### DISCUSSION TITLE
How can i change the APP_URL

### DISCUSSION THREAD

**Discussion by pleibling** [OpnForm/OpnForm#531] [Q&A]
Hello,

i change .env and client/.env with an new APP URL (and restart the container - via docker compose) - but all the times they have the url to http://localhost (also on new forms).

How can i change them?

Thanks for your help.

Regards Peter Leibling

**Comment by JhumanJ** [comment]
Hey Peter, this is a slight configuration issue on our side. please also sent the `FRONT_URL` in the back-end .env. Note that we're about to update the docker setup to mitigate this and few other things
